tests/test_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+import numpy as np
+from tempfile import TemporaryDirectory
+from autowebcompat import utils
+from PIL import Image
+
+
+def test_get_bugs():
+    bugs = utils.get_bugs()
+    assert(isinstance(bugs, list))
+
+
+def test_mkdir():
+    d = TemporaryDirectory()
+    direc_path = os.path.join(d.name + "\\test")
```

Comment:
You're calling `os.path.join` with a single string, which is basically useless.
Either do `os.path.join(d.name, 'test')` or `d.name + '\test'`.